src/components/button/Button.jsx

```diff
@@ -14,7 +14,9 @@ const BUTTON_STYLES = {
   purpleSignUp:
     "h-10 w-40 m-1 p-1 rounded-lg border-2 bg-purple-dark font-light text-[15px] text-white shadow-sm",
   listItem:
-    "h-12 w-72 md:h-14 md:w-80 lg:h-14 lg:w-96 m-1 p-1 rounded-lg border-2 bg-pumpkin font-md text-[17px] text-white shadow-sm",
+    "h-12 w-full md:h-14 lg:h-14 m-1 p-1 rounded-lg border-2 bg-pumpkin font-md text-[17px] text-white shadow-sm",
```

Comment:
in this one specifically i tried to do what you said here but the problem is that there is a very small deference between every button so when u try to merge any repeated classes in the hole buttons you will never find that because for example the rounded and the shadow if you tried to merge thim in one variable you wil change one button style so that this button is not having the same style like the rest 😂